src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,213 @@
+package duke;
+
+import duke.command.DukeException;
+import duke.command.IllegalDateTimeException;
+import duke.command.IllegalDescriptionException;
+import duke.io.ReadFromFile;
+import duke.io.WriteToFile;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.util.ArrayList;
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+
+public class Duke {
+    final static int LENGTH_TODO = 4;
+    final static int LENGTH_DEADLINE = 8;
+    final static int LENGTH_EVENT = 5;
+    final static int LENGTH_BY = 3;
+    final static int LENGTH_AT = 3;
+    final static int SIZE_LINE = 90;
+
+    enum typeOfTasks{
```

Comment:
Good job on using enumerations! However, `enum` types should following the coding standard of using PascalCase. `TypeOfTask` is also a little verbose -- consider changing to `TaskType`.